### ISSUE TITLE
Falling Sequences in Skylanders SWAP Force, SuperChargers, and Imaginators cannot be cleared due to Skylander respawning

### ISSUE THREAD

**Issue by razorchase49** [cemu-project/Cemu#1462]
### Current Behavior

The Skylanders games SWAP Force, Superchargers, and maybe Imaginators each have at least one falling sequence, in which the Skylander must free fall from a great height to reach the bottom and thus proceed with a level. However, when playing with Cemu, the Skylander can only fall for approximately two seconds before they are forcible respawned to the top of the fall, as though they had fallen out of the world. This forced respawning makes these falling sequences impossible to complete, thus making the remainder of the level unbeatable.

### Expected Behavior

In these sequences, the Skylander is expected to reach the bottom of the long fall, collecting coins and avoiding obstacles if there are any. Since the falling is intentional, the Skylander should not have to respawn as if they had fallen out of the world or into areas they shouldn't have.

### Steps to Reproduce

In SWAP Force:
Start the very first level, Mount Cloudbreak, and reach the first caged civilian. Once he is done talking, the floor will break and the dropdown sequence begins. Notice how the Skylander only falls for a few seconds before being respawned to the beginning of the fall, making it impossible to reach the bottom and proceed with the level. There is another level, Twisty Tunnels, that has many similar falling sequences throughout.
In SuperChargers:
Start the Cap'n Cluck's Chicken HQ level (the first part), and reach the part where you encounter Cluck for the first time in the level. After fighting a wave of enemies, he drops you down a hatch, thus starting another falling sequence. Much like the other ones, the Skylander only falls for a few seconds before being respawned to the beginning of the fall.
In Imaginators:
Start the Dragon Temple level, and keep playing until you reach the Alpine Ascent area (you'll know you're there when Air becomes the preferred element) and stumble upon a puzzle involving bridges. Use the first Bounce Pad to reach a ledge that requires a metal bridge. Rather than using the nearby Bounce Pad to get back down, simple walk off the edge to do so. Sometimes, the Skylander can reach the bottom as expected, but usually the Skylander is forced to respawn back at the ledge. This particular case does not make the rest of the level impossible to reach, but it is worth noting, and surely related to the other dropdowns.
A video is attached showing the outcome of each mentioned case.

https://github.com/user-attachments/assets/a5102f66-95ed-4e45-9af4-7f18b0db1b2c

### System Info (Optional)

OS: Windows 11 Pro 64-bit
GPU: NVIDIA GeForce RTX 4070 SUPER

### Emulation Settings (Optional)

Cemu Version: 2.5
Graphics Packs: In each case, I set the framerate to 60fps and resolution to 1920x1080. Also, with SWAP Force, the Portal Stability Fix was enabled.

**Comment by TheRealWario** [comment]
This is a known issue with the fps graphic pack, from what I know. Have you tried disabling them if you are using it?

**Comment by razorchase49** [comment]
Yeah, disabling the FPS pack fixes it. Do you mind sending a link to the bug report for the graphic pack?

**Comment by TheRealWario** [comment]
The graphic packs have their own repository you can report to, although I believe they had fixed it according to the issue someone made: https://github.com/cemu-project/cemu_graphic_packs/pull/639

try updating your graphic packs first, as that should always be the first thing you do when an issue with them comes up.

**Comment by SuperSamus** [comment]
The issue should have already been fixed on all versions of Swap Force and SuperChargers, but not on Imaginators (that game however has other issues that prevent completion at high FPS).

**Comment by kirbeh73** [comment]
This happened to me. If you just stay still and don’t move around to get coins it works.

**Comment by SuperSamus** [comment]
The issue is caused by graphics packs, and has been fixed on that side, so it can be closed. (I can't close issues.)
